pandas/core/frame.py

```diff
@@ -6451,11 +6456,38 @@ def value_counts(
         2         2            0.25
         6         0            0.25
         dtype: float64
+
+        **dropna**
```

Comment:
Don't think this label is specifically needed

Reply:
Removed in f798c5b